roseau/load_flow/models/transformers/transformers.py

```diff
@@ -209,8 +211,11 @@ def _compute_phases_three(
         phases1: str | None,
         phases2: str | None,
     ) -> tuple[str, str]:
-        w1_has_neutral = "y" in parameters.winding1.lower() or "z" in parameters.winding1.lower()
-        w2_has_neutral = "y" in parameters.winding2.lower() or "z" in parameters.winding2.lower()
+        w1 = parameters.winding1
+        w2 = parameters.winding2
+        clock = parameters.phase_displacement
+
+        w1_has_neutral = w1.endswith("N")
         if phases1 is None:
```

Comment:
Should we emit a warning if `n` is in `bus1.phases` but `w1_has_neutral` is False ?